majorityElement.py

```diff
@@ -0,0 +1,18 @@
+class Solution(object):
+    def majorityElement(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        soldierFlag = 0
```

Comment:
Should probably call it `cur`.

Reply:
Also why do you initialise it to `0`.
Should initialise it to `nums[0]` and for your `for` loop, do
```
for num in nums[1:]
```
Of course, you should check `if nums == []` beforehand.